Example/Example-macOS/ContentView.swift

```diff
@@ -0,0 +1,47 @@
+// Created by Cal Stephens on 3/7/24.
+// Copyright © 2024 Airbnb Inc. All rights reserved.
+
+import EpoxyCore
+import SwiftUI
+
+// MARK: - LayoutDemoView
+
+struct LayoutDemoView: View {
```

Comment:
This matches [this demo test case](https://github.com/airbnb/lottie-ios/blob/master/Example/Example/LottieViewLayoutDemoView.swift) we have in the Lottie repo, just using an AppKit `NSImageView` instead of the AppKit `LottieAnimationView`.